CLA_SIGNERS.yaml

```diff
@@ -171,9 +171,6 @@ companies:
       - name: Haikal Pribadi
         email: haikal@grakn.ai
         github: haikalpribadi
-      - name: Borislav Iordanov
-        email: borislav@haikal.ai
-        github: bolerio
       - name: Domenico Corapi
         email: domenico@haikal.ai
```

Comment:
this email address doesn't look right

Reply:
Looks like there are two mistakes (the other one above); both are my fault since I added both of them — I'm guessing the first one was the original mistake, and the most recent one was due to copy-paste-modify. I'll fix this, but I won't fix the one being deleted.

Reply:
Fixed in PR https://github.com/JanusGraph/legal/pull/72